[alexcvzz.vscode-sqlite] sh: 1: preact: not found

```
> vscode-sqlite.htmlcontent@0.0.0 build /tmp/repository/src/resultview/htmlcontent
> preact build --dest ../../../out/resultview/htmlcontent

sh: 1: preact: not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! vscode-sqlite.htmlcontent@0.0.0 build: `preact build --dest ../../../out/resultview/htmlcontent`
npm ERR! spawn ENOENT
```

```
[FAIL] Could not process extension: {
  "id": "alexcvzz.vscode-sqlite",
  "version": "0.8.2",
  "repository": "https://github.com/AlexCovizzi/vscode-sqlite"
}
```

--- a/extensions.json
+++ b/extensions.json
@@ -35,7 +35,7 @@
       "id": "alexcvzz.vscode-sqlite",
       "repository": "https://github.com/AlexCovizzi/vscode-sqlite",
       "version": "0.8.2",
-      "prepublish": "npm install preact-cli"
+      "prepublish": "npm run install-html"
     },
     {
       "id": "aliariff.vscode-erb-beautify",

